hack/ci-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Only run `make ci` if files outside of the site directory changed in the last commit
```

Comment:
The only issue I see with this is that if you have a PR that has 2 commits, the first one containing code and the second one containing docs updates, then `make ci` won't end up getting run since the *last* commit doesn't have code changes.  Not sure off the top of my head if there's an easy way to deal with this.

Reply:
Good point! I've updated it to `git diff` against the destination branch, which should mean it's applicable across all commits being added.